### PR TITLE
Added a few simple documents

### PR DESCRIPTION
**Pull request by ursus-the-bear**

### CODE REVIEW

**Comment by BCNelson** [comment]
Thanks for your contribution. We are currently making some changes  so it could take some time before we are able to review and merge your changes

**Comment by IronicBadger** [comment]
Nice idea but I'm not sure this quite fits with our intent. As said before we're not looking to be a list or apps (that already exists). But instead we'd like to provide specifics to certain apps and provide a level of detail that the awesome list can't.

So if that is possible through changes in this PR then we'll look to merge but as it stands this might be better to start again.

File: docs/self-hosted-apps/git.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Git
+
+As always, check out [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted#project-management) but here are our favourites.
```
Comment (IronicBadger) [inline docs/self-hosted-apps/git.md]: This sentence appears too many times and as we explain in the main PR thread we're not looking to be a list of apps.

**Comment by ursus-the-bear** [comment]
I should have written it in the pull request but did in my mail. I just created placeholder documents until it was clearer what you guys wanted in the documents themselves. I wanted to give you a list of what I could contribute to and would have expanded on the docs once I got a better sense of what you wanted. I’ll just wait until there is more content and then contribute 😊

**Pull request closed without merging**